blockly-core/core/utils/utils.js

```diff
@@ -388,10 +388,22 @@ Blockly.convertCoordinates = function(x, y, svg, toSvg) {
  * @return {!Object} Object with .x and .y properties.
  */
 Blockly.mouseToSvg = function(e, opt_svgParent) {
+  return Blockly.mouseCoordinatesToSvg(e.clientX, e.clientY, opt_svgParent || Blockly.topMostSVGParent(e.target));
```

Comment:
i suspect this is over 80 characters?  Also true of a couple of other places in this PR. We're not super strict about it, but it's good to try to stay under.